tests/specs/web-workers.js

```diff
@@ -0,0 +1,74 @@
+(function (db, describe, it, expect, beforeEach, afterEach) {
+    'use strict';
+    describe('web workers', function () {
+        var dbName = 'tests';
```

Comment:
Instead of this I'm using a guid for the db name, generated using the `beforeEach` test step. Assigning the `dbName` to the `this` scope of `beforeEach` shares it throughout the spec.